### DISCUSSION TITLE
Lock in concerns

### DISCUSSION THREAD

**Discussion by briangladu** [mem0ai/mem0#1679] [General]
Hi!

First of all, thank you for developing this. It looks amazing and is very easy to use and is likely the solution I go with. However, I do have one concern. If I use the hosted cloud version, is there any kind of lock in? What if I need to change solutions later? What will happen to all of my user's "memories"? Will I be able to easily transfer these to a similar provider?

**Comment by deshraj** [comment]
Hey @briangladu, thanks for opening the discussion thread. We are happy to export the memories for your users if needed later.

**Comment by kevin-yauris** [comment]
Hi @deshraj, is this still the case, or is there an alternative way to do it ourselves? I have reviewed the export memory documentation, but I still don't understand it, especially regarding how to utilize the exported memory on the other Mem0 platform account or a self-hosted Mem0. Can you help explain it? Thank you in advance.
